Zotero/Scenes/AppCoordinator.swift

```diff
@@ -33,33 +33,33 @@ protocol AppLoginCoordinatorDelegate: AnyObject {
 
 final class AppCoordinator: NSObject {
     private typealias Action = (UIViewController) -> Void
-
+    
```

Comment:
Careful about these empty lines with tabs, I guess you have it enabled in Xcode so if possible please disable it.